python/lib/dependabot/python/file_updater/pip_compile_file_updater.rb

```diff
@@ -220,7 +220,7 @@ def write_updated_dependency_files
         end
 
         def install_required_python
-          return if run_command("pyenv versions").include?("\ #{python_version}")
+          return if run_command("pyenv versions").include?("\ #{python_version}") # rubocop:disable Style/RedundantStringEscape
```

Comment:
In this case, I disabled the cop because the escaping was intentional to highlight the importance of the leading space: https://github.com/dependabot/dependabot-core/pull/5876#discussion_r994757169.

But I'm think it's maybe better to fix the offense and add a comment, like this?

Reply:
I'm good with the explicit comment approach here.